packages/terra-menu/src/_MenuUtils.js

```diff
@@ -0,0 +1,57 @@
+const getPopupHeight = (contentHeight) => {
+  if (contentHeight <= 40) {
```

Comment:
> Okay. But it's only multiples of 40 up to 160, then it's multiples of 80. I could do an if statement for those two cases which would still greatly reduce the lines of code.

Let's go with this approach. If it reduces it by a lot, I still think it's worth it.